pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
In the case that you don't install or remove dependencies (since there is no changes in `package.json`). Please don't commit the changes in the lock-file!

Reply:
Got it.